src/world/generator/object/TreeType.php

```diff
@@ -34,9 +34,9 @@
  * @method static TreeType DARK_OAK()
  * @method static TreeType JUNGLE()
  * @method static TreeType OAK()
+ * @method static TreeType PLANTED_CRIMSON()
+ * @method static TreeType PLANTED_WARPED()
  * @method static TreeType SPRUCE()
- * @method static TreeType CRIMSON_FUNGUS_PLANTED()
- * @method static TreeType WARPED_FUNGUS_PLANTED()
```

Comment:
You don't need to update this doc comment. It's only here for legacy purposes. New usages of new cases should use the native enum syntax.